appinventor/components/src/com/google/appinventor/components/runtime/util/JavaStringUtils.java

```diff
@@ -140,11 +165,13 @@ public Range(int start, int end, String text) {
   private static final MappingOrder mappingOrderDictionary;
   private static final MappingOrder mappingOrderLongestStringFirst;
   private static final MappingOrder mappingOrderEarliestOccurrence;
+  private static final Comparator<Range> rangeComparator;
 
   static {
     mappingOrderDictionary = new MappingOrder();
```

Comment:
Is there any reason you need these to be initialized in a `static` block rather than just assigning them when the variable is declared?

Reply:
Good point; There's no reason to have it anymore, before that I used it when the code was structured in such a way that required adding things to a data structure. I've removed it.